### ISSUE TITLE
Quelle stratégie autour du produit  ?

### ISSUE THREAD

**Issue by DianeBeldame** [spyrales/gouvdown#38]
En lien avec la prochaine MR #34  (qui formalisera la fin des travaux de notre côté), un peu en lien avec  #issue17, comment s'organise la suite de la longue vie du package ? 
Je m'interroge sur les dimensions suivantes : 
- Qui sont les utilisateurs de {gouvdown} ?
- Quelles difficultés rencontrent-ils ?
- Quelles sont leurs workflows ?
- Pour quel types de publications ? 
- Faut-il, et si oui comment, former les utilisateurs à ce package ? Tant sur le fond que la forme des publications qui en sont issues.

**Comment by RLesur** [comment]
> Qui sont les utilisateurs de {gouvdown} ?

Excellente question. Honnêtement, je ne sais pas trop. Mais lorsqu'on regarde [les personnes qui ont mis une étoile au projet](https://github.com/spyrales/gouvdown/stargazers), {gouvdown} a jusqu'ici bruissé dans la sphère des MEF, des DREAL et d'Etalab.
Je ne saurais dire s'il est utilisé au-delà des DREAL.

Pour les autres questions, j'invite les utilisateurs à réagir ici.

**Comment by MaelTheuliere** [comment]
Quelques réactions : 
- gouvdown est utilisé en Dreal c'est sur, en dehors je ne sais pas j'ai posé une question sur le slack des dreals pour lister les cas d'usage plus précis. Si on fait une recherche de gouvdown sur github on ne croise pas grand chose en tout cas
- pas eu de difficultés remontée, mais j'en vois une : la gestion des logos
- les workvflows concrètement j'en vois trois possibles  : 
  - dans un projet avec un rmd qui produit une publication avec le template et ou les themes graphiques
  - pour produire simplement un graphique type infographie à publier à part
  - dans un package pour produire un template de publi (démarche propre)
- actuellement les seules publications possibles sont des documents html de type bookdown ou rmarkdown, les deux sont utilisés

Et une réaction globale, gouvdown est resté pour l'instant un wip, on avait décidé de communiquer dessus quand on aurait suffisamment avancé. Donc il n' y pas encore vraiment eu une communication active auprès de la communauté cible, c'est à dire les personnes traitant de la donnée dans un service de l'Etat qui utilise la marque Etat. Compte tenu de la volonté du SSP pour l'instant de s'éloigner de ce design guideline, cela réduit la cible actuellement.

**Comment by DianeBeldame** [comment]
Je pensais à quelque chose de plus micro sur les workflow pour identifier les points de frictions, c'est plutôt un usage que tu décris, non ? 

Peux-tu développer @MaelTheuliere  "Compte tenu de la volonté du SSP pour l'instant de s'éloigner de ce design guideline, cela réduit la cible actuellement." ?

**Comment by RLesur** [comment]
@DianeBeldame afin de matérialiser son indépendance fonctionnelle, la statistique publique n'utilise pas la charte État.

**Comment by DianeBeldame** [comment]
Ah. (Denis Brogniart)

**Comment by jdlom** [comment]
Bonjour,
_Réaction d'une personne qui s'intéresse de loin à ce projet. :_

Pour ma part, je suis en DDTM et je ne travaille pas spécialement sur R  pour le moment . 

En revanche, dès que je serai amené à utiliser  R, je suis persuadé que l'utilisation du paquet gouvdown me sera indispensable pour harmoniser la production des documents `made in Etat`

De plus le jour où les modèles de rapport #25 déclinables en différents supports (lettre, présentation, rapport) seront disponibles, cela sera difficile de s'en passer.

**Comment by MaelTheuliere** [comment]
Hello, google m'a mené sur cette page par hasard : https://systeme-de-design.gouv.fr/
Surement un signe ;-) 
Mais voilà à mon avis un endroit où prendre contact, qu'en pensez vous ?

**Comment by DianeBeldame** [comment]
poke @julientaq et @tvroylandt !

**Comment by julientaq** [comment]
Pour l’instant, de ce qu’on peut voir sur le site, le design system de l’État a l’air de se poser surtout sur les questions d’interface et pas vraiment sur les questions de la publication
(de ce qu’on peut en voir).

Il serait intéressant de pouvoir échanger avec l’équipe du *design system* et voir dans quelle dynamique ils se trouvent sur ce point précis, pour que ce système puisse profiter de l’expérience accumulée autour de R, et du savoir faire qui est propre (😉) à vos pratique.

Si c’est possible, j’aimerai bien participer à la conversation :)

**Comment by RLesur** [comment]
Oui, le design system, c'est pour les applications (c'est intéressant pour les gitbooks), ça fait suite à la charte qui elle est plus pour les supports physiques. Mon équipe est en contact avec eux si ça vous intéresse.

**Comment by MaelTheuliere** [comment]
gitbook et shiny @RLesur :-p

**Comment by RLesur** [comment]
Exact, j'oublie toujours shiny, faudrait que j'essaie, une fois 🙂
